src/hopla/hoplalib/hatchery/hatchpotion_data.py

```diff
@@ -35,7 +35,7 @@ class HatchPotionData:
         hopla api content | jq .premiumHatchingPotions | jq keys
     """
 
-    wacky_hatch_potion_names = ["Dessert", "TeaShop", "Veggie", "VirtualPet"]
+    wacky_hatch_potion_names = ["Dessert", "Fungi", "TeaShop", "Veggie", "VirtualPet"]
```

Comment:
Thanks! Welcome to project:+1:

Reply:
Thanks, I will try to resolve more issues.